packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -6,6 +6,7 @@ import 'dart:collection';
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
```

Comment:
material cannot be imported from within material.

Reply:
Stupid IntelliJ!  I've tried turning it off a bunch of times, but it insists on inserting imports when it shouldn't (I didn't add this import manually).